docs/compatibility.md

```diff
@@ -290,10 +290,10 @@ Casting from string to timestamp currently has the following limitations.
  Only timezone 'Z' (UTC) is supported. Casting unsupported formats will result in null values. 
  
 ## UDF to Catalyst Expressions
-To speedup the process of UDF, spark-rapids introduces a udf-compiler extension to translate UDFs to Catalyst expressions.
+To speedup the process of UDF, spark-rapids introduces a udf-compiler extension to translate UDFs to Catalyst expressions. This compiler will be injected automatically to spark extensions by setting `spark.plugins=com.nvidia.spark.SQLPlugin` and is disabled by default.
```

Comment:
nit: I don't think we need this line.  I don't think the user cares about the internal details of how we implemented it, just how to enable it, which is covered by the change below.

Reply:
Right, reasonable to me. Updated.